feature-matcher-generator-e2e-test/src/test/java/ru/yandex/qatools/proc/test/MobTest.java

```diff
@@ -0,0 +1,55 @@
+package ru.yandex.qatools.proc.test;
+
+import org.hamcrest.Matcher;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * @author Vladislav Bauer
+ */
+
+@RunWith(Parameterized.class)
+public class MobTest {
+
+    @Parameterized.Parameters
+    public static Collection<String> matchers() {
+        return Arrays.asList(
+            "withValByte",
+            "withValueByte",
+            "withValChar",
+            "withValueCharacter",
+            "withValInt",
+            "withValueInteger",
+            "withValLong",
+            "withValueLong",
+            "withValFloat",
+            "withValueFloat",
+            "withValDouble",
+            "withValueDouble",
+            "withValBoolean",
+            "withValueBoolean"
+        );
+    }
+
+
+    private final String matcherName;
```

Comment:
please use `@Parameterized.Parameter` instead of constructor. (it looks a little better)